lib/object-mask.js

```diff
@@ -396,34 +405,30 @@ function invert(mask) {
 	return result;
 }
 
-// Throws on (deep) attempt to subtract collection from scalar
 function subtract(a, b) {
 	// base cases:
+	if (a === true) return invert(b);
+	if (a === false) return false;
+	if (!a) return invert(b);
```

Comment:
What does this line do?  You're already checking if `a === false` above, and I don't believe that there's a valid falsy mask other than `false`.  If you want to handle falsy values as `false`, just replace the two above conditionals with `if(a)` and `if(!a)` .